shared/core/src/main/scala/org/hyperledger/identus/shared/messaging/kafka/ZKafkaMessagingServiceImpl.scala

```diff
@@ -80,7 +81,7 @@ class ZKafkaConsumerImpl[K, V](
         .withMaxPollInterval(maxPollInterval) // Should be max.poll.records x 'max processing time per record'
         // 'pollTimeout' default is 50 millis. This is a ZIO Kafka property.
         .withPollTimeout(pollTimeout)
-        // .withOffsetRetrieval(OffsetRetrieval.Auto(AutoOffsetStrategy.Earliest))
+        .withOffsetRetrieval(OffsetRetrieval.Auto(AutoOffsetStrategy.Earliest))
```

Comment:
@bvoiturier, how will this strategy work in the case of the agent restart?
Would it consume all the messages from the beginning?